src/apps/vpn/mozillavpn.cpp

```diff
@@ -319,9 +319,6 @@ void MozillaVPN::initialize() {
   if (m_private->m_user.subscriptionNeeded()) {
     setUserState(UserAuthenticated);
     setState(StateAuthenticating);
-    if (!Feature::get(Feature::Feature_webPurchase)->isSupported()) {
-      TaskScheduler::scheduleTask(new TaskProducts());
```

Comment:
we can remove the taskproducts header. Can we?

Reply:
We still have one Task-Product creation in ``scheduleRefreshDataTasks(bool refreshProducts)``

Reply:
We can remove it. We don't need it right?